src/__tests__/resources/revisions3D.integration.spec.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 Cognite AS
+
+import MockAdapter from 'axios-mock-adapter';
+import { readFileSync } from 'fs';
+import { API } from '../../resources/api';
+import {
+  CreateRevision3D,
+  Model3D,
+  Revision3D,
+  Tuple3,
+  UpdateRevision3D,
+  UploadFileMetadataResponse,
+} from '../../types/types';
+import { setupClient, simpleCompare, retryInSeconds } from '../testUtils';
+
+describe('Revision3d integration test', async () => {
+  let client: API;
+  let mock: MockAdapter;
+
+  beforeAll(async () => {
+    jest.setTimeout(30000);
+    client = await setupClient();
+  });
+
+  const now = Date.now();
+  let revisions: Revision3D[];
+  let file: UploadFileMetadataResponse;
+  let model: Model3D;
+
+  test('create model', async () => {
+    const modelToCreate = { name: `Model revision test ${now}` };
+    model = (await client.models3D.create([modelToCreate]))[0];
+    expect(model).toBeTruthy();
+  });
+
+  test('create file', async () => {
+    const fileContent = readFileSync('src/__tests__/test3dFile.fbx');
+    const fileProps = { name: `file_revision_test_${now}.fbx` };
+    file = await client.files.upload(fileProps, fileContent);
+    expect(file).toBeTruthy();
+  });
+
+  test(
```

Comment:
Same here

Reply:
same as above
but I could merge this two integration tests in one

Reply:
not sure this is a good idea now